autopush/web/base.py

```diff
@@ -255,27 +265,41 @@ def _boto_err(self, fail):
         self._write_response(503, errno=202,
                              message="Communication error, please retry")
 
-    def _router_response(self, response):
+    def _router_response(self, response, router_type=None, vapid=False):
```

Comment:
router_type and vapid should be required (more importantly, webpush and simplepush aren't calling it w/ these args and should be)